boards/nrf52840dongle/Makefile.include

```diff
@@ -0,0 +1,28 @@
+# This board uses the vendor's serial bootloader
+
+PROGRAMMER ?= nrfutil
+
+ifeq (nrfutil,$(PROGRAMMER))
+
+# Using nrfutil implies using the Nordic bootloader described at
```

Comment:
indentation should be 2 spaces in this `ifeq block`

Reply:
Thanks, fixed.